microsoft/testsuites/network/networksettings.py

```diff
@@ -264,3 +264,59 @@ def validate_device_gro_lro_settings_change(self, node: Node, log: Logger) -> No
                 "GRO and LRO settings for all the devices are fixed and cannot be"
                 " changed. Skipping test."
             )
+
+    @TestCaseMetadata(
+        description="""
+            This test case verifies changing device's RSS hash key takes
+            into affect.
+
+            Steps:
+            1. Skip the test if the kernel version is any less than LTS 5.
+            2. Get all the device's RSS hash key values.
+            3. Swap the last 2 characters of original hash key to make a new hash key.
+            4. Validate changing the hash key setting using the new hash key.
+            5. Revert back the settings to original values.
+        """,
+        priority=2,
+    )
+    def validate_device_rss_hash_key_change(self, node: Node, log: Logger) -> None:
+        if node.os.information.version < "5.0.0":
```

Comment:
Here, I have some concern, this condition may good for debian/ubuntu distro.
But the highest kernel version for rhel/centos is 4.X, I think it supports change hkey.